### DISCUSSION TITLE
Generic block tree-sitter textobject

### DISCUSSION THREAD

**Discussion by kirawi** [helix-editor/helix#10550] [Ideas]
I brought this up a while ago on Matrix as [a passing comment](https://matrix.to/#/!zMuVRxoqjyxyjSEBXc:matrix.org/$DEJP6NnbDiCJ0U-bjnJrzsD4vDvbJ45jBn-dyboq8_U?via=matrix.org&via=mozilla.org&via=tchncs.de), but the idea would roughly cover control flow (if/else, loops, match, etc.) or anything else potentially language-specific.

**Comment by pascalkuthe** [comment]
wouldn't that just be covered by `a-o`?
